### DISCUSSION TITLE
Tardis Creation world

### DISCUSSION THREAD

**Discussion by Fristorm-Palifen** [eccentricdevotion/TARDIS#412] [Ideas]
Why not making tardis's world with tardis ID? because with player name we can't abandon it (in my server we have a tardis concession and we can't sell tardis if we can't create some stock xD
![image](https://user-images.githubusercontent.com/15380435/104185959-fec06480-5415-11eb-9949-c0ad49ae2b65.png)

**Comment by Fristorm-Palifen** [comment]
And to prevent spamming abandoned Tardis you can add an int who set the max value of abandoned tardis in a server

**Comment by Fristorm-Palifen** [comment]
and when we delete a Tardis the id will be available for the next new one

**Comment by eccentricdevotion** [comment]
I wouldn't recommend using individual worlds for TARDISes, multiple worlds will start to hog server resources.
If you stick with a default TARDIS world, then abandoning TARDISes and freeing up slots for the next new one is already taken care of.
You can also already restrict the number of TARDISes that players can create.

**Reply by Fristorm-Palifen** [reply]
ok ^^
